client/src/components/reader/ReturnMenuBody.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+import HigherOrder from 'containers/global/HigherOrder';
+
+export default class ReturnMenuBody extends Component {
```

Comment:
I think I mentioned this before — extend PureComponent rather than Component, unless there's a good reason not to.